app/controllers/review_offer.js

```diff
@@ -113,6 +113,13 @@ export default Ember.Controller.extend(AsyncTasksMixin, {
       );
     },
 
+    reopenOffer() {
+      this.runTask(() => {
+        this.get("offerService").reopenOffer(this.get("model"));
+        this.send("toggleOfferOptions");
+      });
+    },
```

Comment:
```javascript
    reopenOffer() {
      this.runTask(async () => {
        await this.get("offerService").reopenOffer(this.get("model"));
        this.send("toggleOfferOptions");
      });
    },
```

If you don't await, the async mixin will think it's over immediately and won't render the spinner correctly

Reply:
sure..i'll add this change.